infra/gcp/clusters/projects/kubernetes-public/aaa/10-cluster-configuration.tf

```diff
@@ -148,6 +148,21 @@ resource "google_container_cluster" "cluster" {
     }
   }
 
+  // Enable NAP
+  cluster_autoscaling {
+    enabled = true
+    resource_limits = [
```

Comment:
I may be wrong, but I think `resource_limits` is a sub-block of `cluster_autoscaling`. You'll need to split the resource types.

Reply:
Fixed, thanks!